tools/build/wheel/split_wheel.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import sys
+import subprocess
+
+import logging
+from pathlib import Path
+from typing import List, Dict, Optional
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+# TODO: Add a function to install dependencies from requirements.txt
+# TODO: Add a function to do submodule setup
+
+
+def get_git_root() -> Path:
+    """
+    Get the current root directory of a Git repository.
+    Returns:
+        str: The path to the Git repository root.
+    """
+    try:
+        # Run the 'git rev-parse --show-toplevel' command
```

Comment:
Do you need this comment? It just repeats what's in the code and will need to be maintained to be in sync.

Reply:
Removed!